<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>380</y>
      <width>581</width>
      <height>191</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>92</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>112</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>100</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>80</y>
      <width>171</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="load_button">
       <property name="accessibleName">
        <string>load_or_save_button</string>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="accessibleName">
        <string>load_or_save_button</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Undo</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>361</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>211</width>
      <height>571</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Dashboard</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>Image Warping</string>
     </property>
     <item>
      <property name="text">
       <string>FishEye Effect</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Swirl Effect</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Waves Effect</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cylinder Anamorphosis</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Radial Blur Effect</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Perspective Mapping</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Custom Effect</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>Image Filtering</string>
     </property>
     <item>
      <property name="text">
       <string>Median Blurring</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gaussian Filtering</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bilateral Filter</string>
      </property>
     </item>
    </item>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
